lib/uri/common.rb

```diff
@@ -419,6 +419,13 @@ def self._decode_uri_component(regexp, str, enc)
   #   URI.encode_www_form({foo: 0, bar: 1, baz: 2})
   #   # => "foo=0&bar=1&baz=2"
   #
+  # The returned string is formed using method URI.encode_www_form_component,
+  # which converts certain characters:
+  #
+  #   URI.encode_www_form('f#o': '/', 'b-r': '$')
+  #   # => "f%23o=%2F&b-r=%24"
+  #
+  #
```

Comment:
I think we can remove this extra newline.

Reply:
Done.